README.md

```diff
@@ -0,0 +1,37 @@
+# covid-sequence-analysis-workflow
+
+This is the official repository of the SARS-CoV-2 variant surveillance pipeline developed under the [Versatile Emerging infectious disease Observatory (VEO)](https://www.globalsurveillance.eu/projects/veo-versatile-emerging-infectious-disease-observatory) project. The project consists of 20 European partners. It is funded by the European Commission.
```

Comment:
Maybe mentioning EMC, ELTE, DTU and EMBL-EBI outright would be good.

'...developed by Danish Technical University (DTU), Eotvos Lorand University (ELTE), EMBL-EBI, Erasmus Medical Center (EMC), under the Versatile...'